followthemoney/graph.py

```diff
@@ -52,22 +66,22 @@ def to_dict(self):
         }
 
     @classmethod
-    def from_proxy(cls, proxy):
+    def from_proxy(cls, proxy: EntityProxy) -> "Node":
         """For a given :class:`~followthemoney.proxy.EntityProxy`, return a node
         based on the entity."""
         return cls(registry.entity, proxy.id, proxy=proxy)
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.caption
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return "<Node(%r, %r, %r)>" % (self.id, self.type, self.caption)
 
-    def __hash__(self):
+    def __hash__(self) -> int:
         return hash(self.id)
 
-    def __eq__(self, other):
-        return self.id == other.id
+    def __eq__(self, other: Any) -> bool:
```

Comment:
Any?

Reply:
That's a suggestion by mypy, pretty weird.